### PR TITLE
Added rpc_status.sh

### PR DESCRIPTION
**Pull request by vyskocilpavel**
* Added script for check if Perun RPC is available

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,6 +98,32 @@ This script checks if the LDAP servers are accessible
...
+    USER=""
+
+    # RPC password
+    PASSWORD=""
+
+    # RPC domain with authentication method
+    # Example: "perun.cesnet.cz/krb"
+    DOMAIN=""
+
+    # Valid userId - This id will be used in getUSerById call
```
Comment (dBucik) [inline README.md]: typo in USer

File: rpc_status.sh [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+USER=""
+
+# RPC password
+PASSWORD=""
+
+# RPC domain with authentication method
+# Example: "perun.cesnet.cz/krb"
+DOMAIN=""
+
+# Valid userId - This id will be used in getUSerById call
```
Comment (dBucik) [inline rpc_status.sh]: typo un USer

File: rpc_status.sh [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+USER_ID=""
+
+# eduPersonPrincipalName which will be searched
```
Comment (dBucik) [inline rpc_status.sh]: ... which the script will look for

File: rpc_status.sh [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+URL="https://${DOMAIN}/rpc/json/usersManager/getUserById?id=${USER_ID}"
+
+START_TIME=$(date +%s%N)
+RPC_RESULT=$(timeout 10 curl --user ${USER}:${PASSWORD} ${URL} 2>&1)
+END_TIME=$(date +%s%N)
+TOTAL_TIME=$(echo "scale=4;$(expr ${END_TIME} - ${START_TIME}) / 1000000000" | bc -l)
+if [[ $RPC_RESULT == *\"id\":${USER_ID}*  ]]; then
+    echo "0 rpc_status total_time=${TOTAL_TIME} OK"
+else
+    echo "2 rpc_status total_time=${TOTAL_TIME} ${RPC_RESULT}"
```
Comment (dBucik) [inline rpc_status.sh:25]: Just a question of concern: is the return code 1 reserved for anything special as you have used 2 instead?

**Review by dBucik — CHANGES_REQUESTED** [review]

File: rpc_status.sh (hunk 4 above)
Comment (vyskocilpavel) [reply rpc_status.sh:25]: The return code 1 is reserved for Nagios state: WARNING

**Review by dBucik — APPROVED** [review]

**Pull request merged**